iml-gui/crate/src/components/date_view.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{
+    components::{attrs, tooltip, Placement},
+    generated::css_classes::C,
+};
+use chrono::{offset::FixedOffset, DateTime};
+use seed::{prelude::*, *};
+
+#[derive(Default)]
+pub struct Model {
+    pub(crate) basedate: Option<DateTime<FixedOffset>>,
+}
+
+impl Model {
+    pub(crate) fn date_view<T>(&self, date: &DateTime<FixedOffset>) -> Node<T> {
```

Comment:
This should be a free fn instead of existing on the model.

Reply:
Yeah, but this function does not show the model, it shows something which does not belong in the model.

Reply:
On the other side it is just syntactic sugar :)